packages/l/libreswan/package.yml

```diff
@@ -21,6 +21,7 @@ builddeps  :
     - xmlto
 conflicts  :
     - strongswan
+    - networkmanager-strongswan
```

Comment:
Why have conflicts for `libreswan` to `networkmanager-strongswan` and `strongswan` to `networkmanager-libreswan`? `networkmanager-strongswan` depends on `strongswan`, and `networkmanager-libreswan` depends on `libreswan`, so adding the conflicts to the `networkmanager` packages should be sufficient to prevent `libreswan` from being installed with `networkmanager-strongswan` (and the other way around).